source/timesheet.js

```diff
@@ -19,23 +21,30 @@ class Timesheet{
         this.notes = timesheet.notes;
     }
 
-    save() {
+    async save() {
         return db.insertOne(this, collectionName);
     }
 
-    delete() {
+    async delete() {
```

Comment:
Likekwise this is unrelated to this issue.

Reply:
Noted - but kept.